services/submission/src/main/java/app/coronawarn/server/services/submission/config/SubmissionServiceConfig.java

```diff
@@ -66,7 +66,7 @@
    *
    * @deprecated should be removed when false
    */
-  @Deprecated
+  @Deprecated(since = "12.08.21", forRemoval = false)
```

Comment:
Can you please use the same format as before?
@Deprecated(since = "2.8", forRemoval = true)